RELEASING.md

```diff
@@ -16,7 +16,18 @@ Version numbers should not be prefixed (e.g. 1.2.3, **not** v1.2.3).
 
 ## Release Procedure
 
-To make a new release, simply run **`npm run release`**.
+**First**, make sure your master branch is up to date and do a dry-run to verify that the expected version number will be used:
+
+```bash
+(master)$ git pull
+(master)$ npm run release -- --dry-run
+```
```

Comment:
What if the dry-run gives an undesired result?

Reply:
Then you should investigate why

Reply:
I meant it would be good to add a link or something how can you manually change the changelog or the version number if needed.

Reply:
We should really *not* do that, everyone should make sure that commit messages are carefully formatted. If the version number is incorrect that is because some commits have not been formatted appropriately.